_posts/2025-01-05-add-a-security-policy.md

```diff
@@ -0,0 +1,43 @@
+---
+layout: post
+published: True
+toc: true
+title: "Add a security policy to your distributions"
+date: 2025-01-05 17:15:00 +0000
+tags: authors guides cpan modules security
+author: Robert Rothenberg
+excerpt: "Adding a SECURITY or SECURITY.md file to your Perl distributions will let people know wow to contact the maintainers if they find a security issue with your software..."
+---
+
+Adding a `SECURITY` or `SECURITY.md`  file to your Perl distributions will let people know:
```

Comment:
Didn‘t we agree on always having an extension, txt, md or pod?

Reply:
I don't think so. But akin to README and LICENSE and CONTRIBUTING, ".txt" is implied. We only add ".md" or ".pod" when it's not vanilla text.